src/components/UserIcon/_user-icon.scss

```diff
@@ -0,0 +1,39 @@
+@import "vanilla-framework/scss/settings_colors";
+
+.user-icon {
+  display: grid;
+  position: relative;
+  text-align: center;
+
+  &__panel {
+    min-width: 100px;
+    opacity: 0;
+    overflow: visible;
+    padding: 0.25rem;
+    position: absolute;
+    right: 0.5rem;
+    top: 2.5rem;
+    transition: opacity 0.25s ease-in-out;
+
+    .p-list {
+      margin-bottom: 0;
+    }
+
+    &.is-visible {
+      opacity: 1;
+      z-index: 1;
+    }
+
+    &-arrow {
+      background: $color-x-light;
+      box-shadow: 3px 3px 6px -3px rgba(0, 0, 0, 0.2);
+      height: 1rem;
+      position: absolute;
+      right: 1.5rem;
+      top: -0.25rem;
+      transform: rotate(225deg);
+      width: 1rem;
+      z-index: 2;
```

Comment:
We should probably implement a z-index stacking order in the application like we had in the old GUI.

https://github.com/canonical-web-and-design/jaas-dashboard/issues/112